packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -886,6 +886,13 @@ export interface DataGridProSharedPropsWithDefaultValue {
    * @default false
    */
   listView: boolean;
+  /**
+   * If set to "always", the multi-sorting is applied without modifier key.
+   * Otherwise, the modifier key is required for multi-sorting to be applied.
+   * @see See https://mui.com/x/react-data-grid/sorting/#multi-sorting
+   * @default "withModifierKey"
+   */
+  multiSortingMode: 'withModifierKey' | 'always';
```

Comment:
```suggestion
  multipleColumnsSortingMode: 'withModifierKey' | 'always';
```

because of https://github.com/mui/mui-x/pull/17925#discussion_r2101269207